### PR TITLE
Add example for delete secret

### PR DESCRIPTION
**Pull request by q384566678**
Signed-off-by: Zhou Hao <zhouhao@cn.fujitsu.com>

### CODE REVIEW

File: cmd/kops/delete_secret.go [hunk 1]
```diff
@@ -34,6 +34,9 @@ var (
 		Delete a secret.`))
 
 	deleteSecretExample = templates.Examples(i18n.T(`
+	# Syntax: kops delete secret <type> <name>
+	# or kops delete secret <type> <name> <id>
+	kops delete secret sshpublickey admin
 
 		`))
```
Comment (hakman) [inline cmd/kops/delete_secret.go:41]:
```suggestion
	`))
```

**Comment by hakman** [comment]
@q384566678 To fix the tests you should run `make gen-cli-docs` and commit the changes.

File: cmd/kops/delete_secret.go (hunk 1 above)
Comment (q384566678) [reply cmd/kops/delete_secret.go]: Done. thanks.

**Pull request merged**